pkgs/by-name/ma/maim/package.nix

```diff
@@ -31,6 +31,7 @@ stdenv.mkDerivation rec {
     hash = "sha256-bbjV3+41cxAlKCEd1/nvnZ19GhctWOr5Lu4X+Vg3EAk=";
   };
 
+  cmakeFlags = [ "-DCMAKE_POLICY_VERSION_MINIMUM=3.10" ];
```

Comment:
Please leave a comment to remind future contributors to remove it once upstream updates it.